### PR TITLE
build: Manually apply lint fixups for import/order

### PR DESCRIPTION
**Pull request by tylerbutler**
We are updating the settings for the import/order lint rule in #20363, so I pre-applied many of the changes from the new settings using the following command:

```shell
pnpm flub exec -g client --concurrency 7 -- 'eslint --fix --fix-type problem,suggestion,layout --rule '"'"'"import/order": ["error", {"newlines-between": "always", "alphabetize": {"order": "asc", "caseInsensitive": false }}]'"'"' src'
```

Note the rule settings used:

```json
"import/order": ["error", {"newlines-between": "always", "alphabetize": {"order": "asc", "caseInsensitive": false }}]
```

I used **case-sensitive sorting** since that's closest to what we currently have.

I did need to do some manual cleanup in tree and devtools-view just to re-arrange some lint disables.

### CODE REVIEW

File: azure/packages/azure-service-utils/src/generateToken.ts [hunk 1]
```diff
@@ -3,11 +3,10 @@
  * Licensed under the MIT License.
  */
 
+import type { ITokenClaims, IUser, ScopeType } from "@fluidframework/protocol-definitions";
 import { KJUR as jsrsasign } from "jsrsasign";
 import { v4 as uuid } from "uuid";
 
-import type { ITokenClaims, IUser, ScopeType } from "@fluidframework/protocol-definitions";
-
```
Comment (DLehenbauer) [inline azure/packages/azure-service-utils/src/generateToken.ts:10]: Expected regrouping

File: azure/packages/test/scenario-runner/src/TestOrchestrator.ts [hunk 2]
```diff
@@ -6,11 +6,10 @@
 import * as fs from "node:fs";
 import path from "node:path";
 
+import { ITelemetryLoggerExt, PerformanceEvent } from "@fluidframework/telemetry-utils";
 import * as yaml from "js-yaml";
 import { v4 as uuid } from "uuid";
 
-import { ITelemetryLoggerExt, PerformanceEvent } from "@fluidframework/telemetry-utils";
-
```
Comment (DLehenbauer) [inline azure/packages/test/scenario-runner/src/TestOrchestrator.ts:13]: Expected regrouping

File: experimental/PropertyDDS/packages/property-properties/src/properties/arrayProperty.js [hunk 3]
```diff
@@ -979,16 +981,16 @@ export class ArrayProperty extends AbstractStaticCollectionProperty {
...
+	 * position: {
+	 * x: 2,
+	 * y: 5
+	 * }
+	 * },
+	 * {
+	 * position: {
+	 * x: 1,
+	 * y: -8
 	 *      }
```
[the remark this answers is not in the document]
Comment (DLehenbauer) [reply experimental/PropertyDDS/packages/property-properties/src/properties/arrayProperty.js]: fixed.

File: experimental/PropertyDDS/packages/property-properties/src/propertyFactory.js [hunk 4]
```diff
@@ -900,15 +893,15 @@ class PropertyFactory {
...
+	 * {
+	 * 'isValid': true or false,
+	 * 'typeid': 'The typeid of the object being parsed',
+	 * 'unresolvedTypes': [ 'An array', 'of strong typeids', 'that were found',
+	 * 'in the document', 'but not resolved from the local cache' ],
+	 * 'resolvedTypes': [ 'Array of', 'strong types resolved', 'during template parsing'],
+	 * 'errors': [ 'Array of', 'objects describing', 'syntax errors in the template' ]
+	 * ...
+	 * }
 	 * </pre>
```
[the remark this answers is not in the document]
Comment (DLehenbauer) [reply experimental/PropertyDDS/packages/property-properties/src/propertyFactory.js]: fixed

File: experimental/PropertyDDS/packages/property-properties/src/properties/setProperty.js [hunk 5]
```diff
@@ -62,10 +63,10 @@ export class SetProperty extends IndexedCollectionBaseProperty {
...
 	 * ```json
 	 * {
-	 *   position: {
-	 *     x: 2,
-	 *     y: 5
-	 *   }
+	 * position: {
+	 * x: 2,
+	 * y: 5
+	 * }
```
[the remark this answers is not in the document]
Comment (DLehenbauer) [reply experimental/PropertyDDS/packages/property-properties/src/properties/setProperty.js]: fixed.

File: experimental/PropertyDDS/packages/property-properties/src/propertyFactory.js [hunk 6]
```diff
@@ -2290,16 +2283,16 @@ class PropertyFactory {
...
+	 * errors: {
+	 * typeid1: errors,
+	 * ...
+	 * typeidn: errors
+	 * },
+	 * templates: {
+	 * typeid1: [], array of templates
+	 * ...
+	 * typeidn: []  array of templates
+	 * }
```
[the remark this answers is not in the document]
Comment (DLehenbauer) [reply experimental/PropertyDDS/packages/property-properties/src/propertyFactory.js]: fixed

File: packages/drivers/routerlicious-driver/api-report/routerlicious-driver.api.md [hunk 7]
```diff
@@ -80,7 +80,10 @@ export const RouterliciousErrorTypes: {
     readonly genericNetworkError: "genericNetworkError";
     readonly authorizationError: "authorizationError";
     readonly fileNotFoundOrAccessDeniedError: "fileNotFoundOrAccessDeniedError";
-    readonly offlineError: "offlineError";
+    readonly offlineError: "offlineError"; /**
+    * Interface for error responses for the WebSocket connection
+    * Intended to be compatible with output from `NetworkError.toJSON`.
+    */
```
[the remark this answers is not in the document]
Comment (DLehenbauer) [reply packages/drivers/routerlicious-driver/api-report/routerlicious-driver.api.md]: fixed

**Review by DLehenbauer — APPROVED** [review]

**Review by DLehenbauer — APPROVED** [review]

**Review by noencke — APPROVED** [review]

**Review by Josmithr — APPROVED** [review]
Devtools changes look good

**Review by Abe27342 — APPROVED** [review]

**Review by alexvy86 — APPROVED** [review]

**Review by alexvy86 — APPROVED** [review]
Fluid Devtools looks good

**Pull request merged**